Modules/Microsoft365DSC/DSCResources/MSFT_AADGroup/MSFT_AADGroup.psm1

```diff
@@ -1088,9 +1084,9 @@ function Test-TargetResource
         try
         {
             if ($null -ne $CurrentValues.AssignedLicenses -and $CurrentValues.AssignedLicenses.Length -gt 0 -and `
-                $null -eq $AssignedLicenses)
+                ($PSBoundParameters.ContainsKey('AssignedLicenses') -and $null -eq $AssignedLicenses))
             {
-                Write-Verbose -Message "The group currently has licenses assigned but it shouldn't"
+                Write-verbose -Message "The group {$DisplayName} currently has licenses assigned but it shouldn't"
```

Comment:
```suggestion
                Write-Verbose -Message "The group {$DisplayName} currently has licenses assigned but it shouldn't"
```
Very minor: You got some typos in `Write-Verbose` statements.

Reply:
Please explain. I don't see them

Reply:
Unless you mean Write-Verbose (with a capital 'V') instead of Write-verbose (with a lower-case 'v')

Reply:
Yes exactly. We should always respect the correct capital or lower case, even though PowerShell doesn't care.